docs/validators/faq.mdx

```diff
@@ -0,0 +1,57 @@
+---
+description: Frequently asked questions about Connect.
+title: FAQs
+icon: circle-question
+---
+
+<AccordionGroup>
+    <Accordion title="Can I run Connect on the same machine as my validator?">
+        Yes, you can run it anywhere - but please defer to any chain-specific recommendations if there are any!
+    </Accordion>
+
+    <Accordion title="Can I use the same Connect sidecar for different chains?">
+        We are currently working on supporting this feature. It will be available in a future release.
+    </Accordion>
+
+    <Accordion title="Does Connect store historical prices?">
+        No. Prices are stored in `x/oracle` module, and only stores the most recently posted price. However, you can use blockchain indexers or inspect past blocks to see the prices committed on previous heights.
+    </Accordion>
+
+    <Accordion title="How often are prices updated on-chain?">
+        Prices within Connect are committed on a one-block delay, since validators use the vote extensions from block n-1 to securely submit their price data for block n.
+
+        Most of the time, prices will update every single block. Price updates happen when there is a significant change in price, and over 66% of validators are correctly running the Connect sidecar.
```

Comment:
For anything besides dydx don't we always update the price even if over 2/3 of vals are behind it?

The significant change part is only because of dydx min price change ppm right?

Reply:
[remove the ppm callout in faq](https://github.com/skip-mev/slinky/pull/604/commits/8df7a9467a103dd309210bb5e080cf86476e29ab)

Reply:
yes